src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,24 @@
+{% trans_default_domain "locationview" %}
+
+{% if draftVersions %}
+    <div class="ez-list-toolbar">
+        <h2 class="ez-list-toolbar-label">{{ 'locationview.versions.draft.under.edit'|trans()|desc('Draft under edit') }}</h2>
+    </div>
+    {% include '@EzSystemsHybridPlatformUi/content/tabs/versions/table.html.twig' with {'versions': draftVersions} %}
```

Comment:
According to knowledgeable Symfony users, `{{ include() }}` should be used instead of `{% include %}`: https://twig.sensiolabs.org/doc/2.x/functions/include.html.